lib/api/response_decoder.dart

```diff
@@ -4,4 +4,7 @@ class ResponseDecoder {
   static Map<String, dynamic> decode(Map<String, dynamic> json) {
     return Japx.decode(json)['data'];
   }
+
+  static Map<String, dynamic> decodeAtRoot(Map<String, dynamic> json) =>
```

Comment:
I think we should change this function to `decode`, and the old one above is `decodeData`
```
static Map<String, dynamic> decodeData(Map<String, dynamic> json) => decode(json)['data']

```

Reply:
Good suggestion 👍 , I updated it at https://github.com/nimblehq/flutter-ic-khanh-thieu/pull/58/commits/d7c069b2ba52a8803171c900fe495ca646377ca1 🙏